src/test/operators/print_test.cpp

```diff
@@ -181,52 +179,6 @@ TEST_F(OperatorsPrintTest, TruncateLongValueInOutput) {
   EXPECT_TRUE(output_string.find(manual_substring) != std::string::npos);
 }
 
-TEST_F(OperatorsPrintTest, EmptyChunkFlag) {
-  // Start off by adding an empty chunk
-  auto tab = StorageManager::get().get_table(_table_name);
-  Segments empty_segments;
-  auto column_1 = std::make_shared<opossum::ValueSegment<int>>();
-  auto column_2 = std::make_shared<opossum::ValueSegment<pmr_string>>();
-  empty_segments.push_back(column_1);
-  empty_segments.push_back(column_2);
-  tab->append_chunk(empty_segments);
-
-  auto wrap = std::make_shared<TableWrapper>(tab);
-  wrap->execute();
-
-  // Flags = 0 is the default. As such, empty chunks will be printed.
-  std::ostringstream output_withempty;
-  auto print_wrap_withempty = PrintWrapper(wrap, output_withempty, 0);
-  print_wrap_withempty.execute();
-
-  auto expected_output_withempty =
-      "=== Columns\n"
-      "|column_1|column_2|\n"
-      "|     int|  string|\n"
-      "|not null|not null|\n"
-      "=== Chunk 0 ===\n"
-      "Empty chunk.\n";
```

Comment:
Might be related to my understanding question: we still print the empty information, but apparently no test does cover this situation?